.eslintrc

```diff
@@ -1,12 +1,22 @@
 {
     "extends": [
         "finn",
-        "finn/node"
+        "finn/node",
+        "finn-prettier"
```

Comment:
I general yes that would be preferable, but I don't want to burden @sveisvei with to much extra work to separate it in it's own PR...